style.css

```diff
@@ -469,36 +482,50 @@
     border-radius: 10px;
     margin-top: 32px;
   }
+
+  .ruby {
+    display: none;
+  }
 }
 
-/* responsiveness */
+/* responsiveness RWD for tablets */
 
-@media only screen and (min-width: 700px) {
+@media only screen and (max-width: 1200px) and (min-width: 769px) {
```

Comment:
- [ ] [optional] It would be better just to use one breakpoint for your desktop version since you don't have a different version for different desktop screens.